test/fixtures/output/python/python3/application-json.py

```diff
@@ -0,0 +1,15 @@
+import http.client
+
+conn = http.client.HTTPConnection("mockbin.com")
+
+payload = "{\"number\": 1, \"string\": \"f\\\"oo\", \"arr\": [1, 2, 3], \"nested\": {\"a\": \"b\"}, \"arr_mix\": [1, \"a\", {\"arr_mix_nested\": {}}] }"
```

Comment:
It's the exact same as all the other fixtures, see [node for example](https://github.com/ahmadnassri/httpsnippet/blob/master/test/fixtures/output/node/native/application-json.js#L26)

Anyways I'm running the following to escape the payload like the other targets, maybe this should be abstracted away as a helper.

    JSON.stringify(source.postData.text)